ee/packages/pdf-worker/src/strategies/ChatTranscript.ts

```diff
@@ -86,7 +121,7 @@ export class ChatTranscript implements IStrategy {
 					.format('H:mm:ss')} ${data.timezone}`,
 			},
 			messages: Array.isArray(data.messages)
-				? this.parserMessages(data.messages, data.dateFormat as string, data.timeAndDateFormat as string, data.timezone as string)
+				? await this.parserMessages(data.messages, data.dateFormat as string, data.timeAndDateFormat as string, data.timezone as string)
```

Comment:
as this is now async and the number of messages in a conversation could be really big, i wonder if we should start batching these 🤔 

wdyt @MarcosSpessatto ?

Reply:
For sure, I agree with that.

Reply:
Not needed anymore, as made the process synchronous.